jiant/utils/data_loaders.py

```diff
@@ -113,7 +113,8 @@ def load_tsv(
         s2_idx (int|None): The column index for the second text field, if present.
         label_fn (fn: str -> int|None): A function to map items in column label_idx to int-valued
             labels.
-        col_indices (list[int]): TODO(YADA): ???
+        col_indices (list[int]): column names, which should be 1-n, where n is the number of columns
```

Comment:
@pruksmhc Sorry to keep bugging you about this, but I need more help to figure out what to do here. I still don't understand what this is, or when it's used. Why is a list of indices called 'names', and why is it required when we already give the indices of the text and labels?

Reply:
Regardless of the column names, we currently index based on integers (since we specify indices as integers). Thus, we need to pass in col_indices. Another way to actually do this is to change the default value of skip_rows to 0, then we don't need to pass in col_indices. I'll make that change to make it clearer.

Reply:
Did you mean to push something? I still don't really understand from this comment. We already give column indices in the previous argument. What indices are these? Where are they used?

Reply:
Thanks!